CHANGELOG.md

```diff
@@ -10,12 +10,14 @@ accidentally triggering the load of a previous DB version.**
 * #5241 Allow RETURNING clause when inserting into compressed chunks
 * #5245 Mange life-cycle of connections via memory contexts
 * #5246 Make connection establishment interruptible
+* #5262 Extend enabling compression on a continuous aggregrate with 'segmentby' and 'orderby' parameters
```

Comment:
Maybe something like "allow to specify `compress_segmentby` and `compress_orderby` parameters when enabling a compression on a continuous aggregate table"?